packages/base/src/3dview/mainview.tsx

```diff
@@ -685,6 +686,7 @@ export class MainView extends React.Component<IProps, IStates> {
       wireframe: this.state.wireframe
     });
     this._clippingPlaneMesh = new THREE.Mesh(planeGeom, planeMat);
+    this._clippingPlaneMesh.visible = this._clipSettings.enabled;
```

Comment:
Do we still need this now that we set `stencil: true` on the renderer?

I'm fine letting this here though, it does sound sensible to not render the plane in anyway when the clipping is disabled.

Reply:
^^ Just a question, not a blocker

Reply:
> Do we still need this now that we set `stencil: true` on the renderer?

Yes, it's necessary to be there as it avoids the clip plane mesh from rendering itself fully and renders only the part of the plane which intersect with the geometry. 

It avoids this from happening: 

<img width="335" alt="image" src="https://github.com/user-attachments/assets/6752bed2-6ec8-4ee7-b9f1-7e5e6d258b95">

Reply:
Ok then let's keep it! I'm unsure what changed in ThreeJS that requires this change, but the code looks more sensible now so that's good I guess